docs/life_of_a_godwoken_transaction.md

```diff
@@ -66,11 +69,11 @@ Godwoken provides an account model to Nervos CKB:
 * A 256-bit key-value storage is provided to applications running on Godwoken.
 * Conceptually, Godwoken provides a sequential model: each layer 2 transaction is applied to the Godwoken storage in sequence. Smart contracts in layer 2 transactions are free to read from and write into the storage space.
 
-One might wonder where all the data are stored given constant storage requirement in the above transactions. Godwoken leverages [Sparse Merkle Tree](https://medium.com/@kelvinfichter/whats-a-sparse-merkle-tree-acda70aeb837) to build the storage space. Only the root hash of the Sparse Merkle Tree(SMT) is kept in the rollup cell. State validator validates that the sparse merkle tree is correctly updated in each layer 2 block. We actually took a step further here, and build an optimized SMT, which can save a lot of space and calculation for CKB's use case. Please refer to [here](https://github.com/jjyr/sparse-merkle-tree) for more details.
+One might wonder where all the data are stored given constant storage requirement in the above transactions. Godwoken leverages [Sparse Merkle Tree](https://medium.com/@kelvinfichter/whats-a-sparse-merkle-tree-acda70aeb837) to build the storage space. Only the root hash of the Sparse Merkle Tree(SMT) is kept in the rollup cell. State validator validates that the sparse merkle tree is correctly updated in each layer 2 block. We actually took a step further here, and build an optimized SMT, which can save a lot of space and calculation for CKB's use case. Please refer to [here](https://github.com/nervosnetwork/sparse-merkle-tree) for more details.
```

Comment:
Considering the constant storage requirements of the above transactions, you may wonder where all the data is stored. Godwoken leverages [Sparse Merkle Tree](https://medium.com/@kelvinfichter/whats-a-sparse-merkle-tree-acda70aeb837) to build the storage space. The rollup cell only contains the root hash of the Sparse Merkle Tree (SMT). The state validator verifies that the sparse merkle tree is correctly updated in each layer 2 block. We have actually gone one step further here, and built an optimized SMT, which will save CKB a great deal of space and calculation. For more details, see [here](https://github.com/nervosnetwork/sparse-merkle-tree).